build/patch-charts.sh

```diff
@@ -155,10 +154,12 @@ function patchGalley() {
 
   convertClusterRoleBinding "${HELM_DIR}/istio-control/istio-discovery/templates/clusterrolebinding.yaml" ""
   sed_wrap -i -e '/- "discovery"/ a\
+{{- if not .Values.global.clusterScoped }}\
           - --memberRollName=default\
+          - --disableNodeAccess=true\
+{{- end}}\
           - --enableCRDScan=false\
-          - --enableIngressClassName=false\
-          - --disableNodeAccess=true' "${deployment}"
+          - --enableIngressClassName=false' "${deployment}"
```

Comment:
same here, this can be removed for cluster-scoped mode, right?

Reply:
Done.